README.md

```diff
@@ -10,4 +10,239 @@ TODO: description
+
+**Body**
+
+- `subscriber_name` _{string}_ - The insurance card's subscriber
+- `member_id` _{string}_ - The insurance card's member ID
+- `group_number` _{string}_ - The insurance card's group number
+- `plan_number` _{string}_ - The insurance card's plan number
+- `plan_type` _{string}_ - The insurance card's plan type
+- `purpose` _{string}_ - The insurance card's purpose
+- `notes` _{string}_ - Notes about the insurance card
+
+**Returns**
+
+- A success message
+- A object with the created insurance card
+
+**Throws**
+
+- `403` if the user is not logged in
+- `400` If the subscriber_name or purpose is empty or a stream of empty spaces
+
+#### `PATCH /api/insuranceCards/:insuranceCardId?` - Edit the information of an insurance card
+
+**Body**
+
+- `subscriber_name` _{string}_ - The insurance card's subscriber
+- `member_id` _{string}_ - The insurance card's member ID
+- `group_number` _{string}_ - The insurance card's group number
+- `plan_number` _{string}_ - The insurance card's plan number
+- `plan_type` _{string}_ - The insurance card's plan type
+- `purpose` _{string}_ - The insurance card's purpose
+- `notes` _{string}_ - Notes about the insurance card
+
+**Returns**
+
+- A success message
+- A object with the edited insurance card
+
+**Throws**
+
+- `403` if the user is not logged in
+- `400` If the subscriber_name or purpose is empty or a stream of empty spaces
+
+#### `DELETE /api/insuranceCards/:insuranceCardId?` - Delete an insurance card
+
+**Returns**
+
+- A success message
+
+**Throws**
+
+- `403` if the user is not logged in
+- `403` if the user is not the owner of the insurance card
+- `404` if the insuranceCardId is invalid
+
+### Medical Contact
+
+#### `GET /api/medicalContacts?owner=USERNAME` - Get medical contacts by owner
+
+**Returns**
+
+- An array of medical contacts created by user with username `owner` in alphabetical order with respect to the last name of the contact
+
+**Throws**
+
+- `400` if `owner` is not given
+- `404` if `owner` is not a recognized username of any user
+
+#### `POST /api/medicalContacts` - Create a new medical contact
+
+**Body**
+
+- `active` _{string}_ - Whether the medical contact is in use or not
+- `title` _{string}_ - The medical contact's title
+- `first_name` _{string}_ - The medical contact's first name
+- `last_name` _{string}_ - The medical contact's last name
+- `hospital` _{string}_ - The medical contact's associated hospital
+- `specialty` _{string}_ - The medical contact's specialty
+- `phone_number` _{string}_ - The medical contact's phone number
+- `notes` _{string}_ - Notes about the medical contact
+
+**Returns**
+
+- A success message
+- A object with the created medical contact
+
+**Throws**
+
+- `403` if the user is not logged in
+- `400` If the first_name, last_name, or phone_number is empty or a stream of empty spaces
+
+#### `PATCH /api/medicalContacts/:medicalContactId?` - Edit the information of a medical contact
```

Comment:
same comment as the patch above: do we want to require all fields or just set those provided?